build.gradle

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Actually this is not newly added, I updated the existing gradle file.